packages/home-spa/src/mocks/stub.js

```diff
@@ -2,7 +2,7 @@ export const stub = [
    {
       name: "Authentication",
       description: "Adding authentication to your application is pice of cake. Just integrate with our API and focus on application development.",
-      link: "#",
+      link: "microservice-details.html#Authentication",
```

Comment:
1. link needs to be small-caps.
2. replace space with "-" in-bitween words.

Reply:
Made the changes